pkg/client/cli/cmds_intercept.go

```diff
@@ -151,6 +159,13 @@ func interceptCommand(ctx context.Context) *cobra.Command {
 
 	flags.StringVarP(&args.namespace, "namespace", "n", "", "If present, the namespace scope for this CLI request")
 
+	flags.BoolVarP(&args.cmdLineIngress, "cmd-ingress", "i", false, "if set, uses cmd flags instead of opening a dialogue for ingress settings."+
```

Comment:
Why isn't this flag implied by the other ingressXXX flags?

Reply:
I agree with Thomas here. I think instead of having `cmdLineIngress` as a flag, just add the others, and if the user has already specified the flag, don't call the `askfor*` function inside of `selectIngress` function and use the value provided by the user.  (you could add some output like: "Using value for x" so the user knows why we aren't asking them to provide input.

Reply:
Yes, I will get rid of the -i flag and check if any of the other flags are set.

I dont really want to make the flags answer individual questions in "selectIngress" because I don't think that is a use-case. I dont think anyone would ever want to only skip half the dialogue questions.